distribution/docker/README.md

```diff
@@ -24,3 +24,15 @@ From the root of the repo, run `docker build -t druid:tag -f distribution/docker
 ## Run
 
 Edit `environment` to suite. Run 'docker-compose -f distribution/docker/docker-compose.yml up`
+
+## mysql database connector
+
+This image contains solely the postgres metadata database connector. If you need
+the mysql metadata storage connector, consider adding these lines before the `addgroup`
```

Comment:
I forget the details of the original conversation, but would it be possible to add a `Dockerfile.mysql` or similar that uses a FROM clause against the other docker container and just runs this command?

That way the instructions can be something like "Build the dockerfile at `something/something/Dockerfile.mysql` and specify the upstream tag like this: `docker build -t sometag -f something/something/Dockerfile.mysql`" or similar?

Reply:
I don't think this is worth doing. Nonetheless I've added it

most people want the built image from the repo.

for those that don't, they may want other changes in here, or different version. I think what is here satisfies most, e.g. the instructions are there, and the docker image will get built and published (which is the main goal).